test-data/unit/check-generic-subtyping.test

```diff
@@ -99,8 +99,11 @@ class F: pass
 from typing import TypeVar, Generic
 t = TypeVar('t')
 class I(Generic[t]): pass
-class A(I[C], I[object]): pass # E: Duplicate base class "I"
+class A(I[C], I[object]): pass
 class C: pass
+[out]
+main:4: error: Duplicate base class "I"
+main:4: error: Cannot determine consistent method resolution order (MRO) for "A"
```

Comment:
This extra error message is actually disturbing. It comes from ThirdPass. I'm going to meditate on the right solution some more.